packages/@lwc/engine-core/src/framework/base-bridge-element.ts

```diff
@@ -152,6 +152,13 @@ export function HTMLBridgeElementFactory(
 
     // To avoid leaking private component details, accessing internals from outside a component is not allowed.
     descriptors.attachInternals = {
+        set() {
+            if (process.env.NODE_ENV !== 'production') {
+                logError(
```

Comment:
Let's use `logWarn` from now on for dev-only console stuff. Otherwise it confuses people when debugging some unrelated error.